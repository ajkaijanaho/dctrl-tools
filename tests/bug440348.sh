#!/bin/sh

set -e

$SORT_DCTRL < 0006.in
