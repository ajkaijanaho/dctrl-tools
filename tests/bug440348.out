Package: abiword-plugins-gnome
Status: purge ok not-installed
Priority: optional
Section: gnome

